content/en/docs/rate-limits.md

```diff
@@ -3,131 +3,140 @@ title: Rate Limits
+
+Limit: 300 new orders / ACME account / 3 hours
+
+If a new order is requested and there already exists a pending order covering the exact same
+<a href="#FQDN">FQDNs</a>, the pending order will be returned and a new order will **not**
+be created.
+
+***
+
+Message: too many registrations for this IP
+
+Limit: 10 ACME account registrations / IP address / 3 hours
+
+***
+
+Message: too many registrations for this IP range
+
+Limit: 500 ACME account registrations / IPv6/48 range / 3 hours
+
+***
+
+Message: too many currently pending authorizations
+
+Limit: 300 pending authorizations / ACME account
+
+Hitting this limit is usually the result of a malfunctioning ACME client.
+If you know the URL of a pending authorization of the form `/acme/authz/XYZ`, which can likely
+be found in the log of your ACME client, you can trigger a validation attempt for that
+authorization by submitting a JWS-signed POST request to any of its challenges as described in
+the [ACME specification](https://tools.ietf.org/html/rfc8555#section-7.5.1). Regardless of the
+outcome of the validation attempt, the authorization will no longer be pending.
+
+***
+
+Limit: 20 requests / second
+
+Hitting this limit is usually the result of a malfunctioning ACME client.
+This limit applies collectively to requests for the "new-reg", "new-authz", and "new-cert"
+ACMEv1 API endpoints and the "new-nonce", "new-account", "new-order", and "revoke-cert"
+ACMEv2 API endpoints.
+
+***
+
+Limit: 40 requests / second
+
+Hitting this limit is usually the result of a malfunctioning ACME client.
+This limit applies collectively to requests for the "/directory" endpoint and the "/acme" 
+directory and subdirectories.
+
+# Definitions
+
+<span id="duplicate-certificate">Duplicate Certificate</span>
+
+A certificate is considered a duplicate if it contains the exact same <a href="#SAN">SANs</a>
+as another certificate, regardless of order. Renewal certificates are duplicate certificates.
+
+***
+
+<span id="FQDN">Fully Qualified Domain Name (FQDN)</span>
+
+An FQDN is a complete domain name consisting of any subdomain names and an
+<a href="#apex-domain-name">apex domain name</a>.
+
+***
+
+<span id="apex-domain-name">Apex Domain Name</span>
+
+An apex domain name is usually the trailing part of an <a href="#FQDN">FQDN</a> registered through
+a registrar. For instance, `example.com` is the apex domain name of `www.example.com`.
+We use the [Public Suffix List](https://publicsuffix.org) to determine the apex domain
+name for any given <a href="#FQDN">FQDN</a>.
+
+***
+
+<span id="SAN">Subject Alternative Name (SAN)</span>
+
+Each <a href="#FQDN">FQDN</a> that a certificate covers is listed as a SAN in the certificate.
+A certificate with multiple SANs is sometimes called a unified communications certificate (UCC).
+Reducing the number of SANs in a certificate reduces the operational complexity associated with
+using the certificate, resulting in increased performance and reliability.
```

Comment:
This seems a bit editorial, on the subject of having more certs / fewer names/cert being more performant. It reduces the size, though, which makes handshakes smaller, which would be a noncontroversial remark.

Reply:
This is taken directly from the currently-published page:
"Note: For performance and reliability reasons, it’s better to use fewer names per certificate whenever you can."

I just wrote "operational complexity" to cover broad ground, including, but not mentioned, such complexities as revocation of a certificate loaded with a massive number of SANS causing operational disaster. This is akin to the situation of a single certificate with multiple SANs for domains being hosted on separate machines sharing a single private key. The practical operational complexity (and security risk) associated with managing a shared private private key is greater than simply using multiple certificates with fewer SANs.

I guess I just don't see anything especially controversial about my revision:
"Reducing the number of SANs in a certificate reduces the operational complexity associated with using the certificate, resulting in increased performance and reliability."

Reply:
I simply remember the prior uphill battles about ACME automation encouraging individual certs running into older automations encouraging one cert per host.  I don't want my nitpicking over battles long since won to hold up the rest of this very excellent change, but I'm also not sure I should be approving here, so tagging @aarongable for final review.

Reply:
Aaron already knows and has let me know that some others want to review too. Thanks for taking the time and giving good input. 😊